src/js/logic/pdf-to-bmp.ts

```diff
@@ -59,33 +60,28 @@ export async function pdfToBmp() {
     const pdf = await getPDFDocument(
       await readFileAsArrayBuffer(state.files[0])
     ).promise;
-    const zip = new JSZip();
 
-    for (let i = 1; i <= pdf.numPages; i++) {
-      showLoader(`Processing page ${i} of ${pdf.numPages}...`);
-      const page = await pdf.getPage(i);
-      const viewport = page.getViewport({ scale: 1.5 });
-      const canvas = document.createElement('canvas');
-      const context = canvas.getContext('2d');
-      canvas.height = viewport.height;
-      canvas.width = viewport.width;
+    if(pdf.numPages === 1) {
+        showLoader(`Processing page 1 of ${pdf.numPages}...`);
```

Comment:
this would be always page 1 of 1 and could be named something like... single page conversion